test/adze.ts

```diff
@@ -233,3 +233,21 @@ test('global filter includes logs based on namespace', (t) => {
   t.falsy(d_printed);
   t.falsy(v_printed);
 });
+
+test('namespace allows rest parameters', (t) => {
```

Comment:
This test should be written for both the browser environment and node environment under their respective folders in the test file named `test/[browser|node]/modifiers/identifying.ts`. You will notice they differ slightly in the number of "args" that exist in the log render which is why they are split between each environment.

Reply:
Oh right! my bad, didn't have spotted specific test files for namespace. I already moved them to their respective folders